src/main/java/com/jcabi/ssh/SSH.java

```diff
@@ -68,6 +67,9 @@
  * <p>It is highly recommended to use classes from {@link Shell} interface,
  * they will simplify operations.</p>
  *
+ * @todo This class has common data, implementation of exec() and
```

Comment:
@Suseika adding a puzzle you have to add ticket number and estimation, otherwise it couldn't be merged. In this case it should look like: `@todo #21:30min ...`